tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/async/JaxRsClientAsyncTestEndpoint.java

```diff
@@ -69,16 +71,18 @@ private void closeClient() {
 
     @GET
     @Path("/jaxrsclient")
-    public Response getJax(@Context UriInfo uriInfo) {
+    public Response getJax(@Context UriInfo uriInfo, @QueryParam(value = "baggageValue") String baggageValue) {
         Assert.assertNotNull(Span.current());
 
-        try (Scope s = Baggage.builder().put("foo", "bar").build().makeCurrent()) {
+        try (Scope s = Baggage.builder().put("foo", baggageValue).build().makeCurrent()) {
             Baggage baggage = Baggage.current();
-            Assert.assertEquals("bar", baggage.getEntryValue("foo"));
+            Assert.assertEquals(baggageValue, baggage.getEntryValue("foo"));
 
             String url = new String(uriInfo.getAbsolutePath().toString());
-            url = url.replace("jaxrsclient", "jaxrstwo"); // The jaxrsclient will use the URL as given so it needs
-                                                          // the final part to be provided.
+            url = url.replace("jaxrsclient", "jaxrstwo") + "?baggageValue=" + baggageValue; // The jaxrsclient will use
+                                                                                            // the URL as given so it
+                                                                                            // needs
+            // the final part to be provided.
 
             String result = client.target(url)
                     .request(MediaType.TEXT_PLAIN)
```

Comment:
Rather than doing this, you can add the query parameter when you make the request

```java
            String result = client.target(url)
                    .request(MediaType.TEXT_PLAIN)
                    .queryParameter("baggageValue", baggageValue)
                    .get(String.class);
```